.github/workflows/client-tests.yaml

```diff
@@ -0,0 +1,25 @@
+name: Client tests
```

Comment:
Looks like you might have created this branch off develop? Could you rebase off the release branch 🙏

Reply:
Yes! I have rebased from release, however I think develop had additional changes which haven't been included in the release branch.

James and Mark reckoned that these changes would be OK to include in release branch, so I've left them in here rather than making a new branch.